build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1311,7 +1311,8 @@ message GetActionResultRequest {
   bool inline_stderr = 4;
 
   // A hint to the server to inline the contents of the listed output files.
-  // Each path needs to exactly match one path in `output_files` in the
+  // Each path needs to exactly match one path in `output_paths` or
```

Comment:
nitpick: I guess it would be more precise to say that it needs to match one _file path_ in either of these fields?